weave/scorers/llm_utils.py

```diff
@@ -2,10 +2,6 @@
 
 from typing import TYPE_CHECKING, Any, Union
 
-from weave.trace.autopatch import autopatch
-
-autopatch()  # ensure both weave patching and instructor patching are applied
```

Comment:
this seems out of place originally - might be good to check with Morgan why this was added here.